es/client.go

```diff
@@ -44,14 +45,15 @@ func init() {
 	for r := retryCount; r > 0; r-- {
 		Client, err = elastic.NewClient(options...)
 		if err != nil {
-			logger.Warning(fmt.Sprintf("Failed to connect to ElasticSearch database. Retrying in %s seconds.", retrySeconds), err.Error())
+			logger.Warning(fmt.Sprintf("Failed to connect to ElasticSearch database. Retrying in %v seconds.", retrySeconds), err.Error())
 			time.Sleep(retrySeconds * time.Second)
 		} else {
 			logger.Info("Successfully connected to ElasticSearch database.", nil)
 			return
 		}
 	}
 	logger.Error("Failed to connect to ElasticSearch database. Not retrying.", nil)
+	log.Fatal("Exiting due to connection fail.")
```

Comment:
Use `logger` not `log`

Reply:
Changed to `os.Exit(1)` because logger doesn't support fatal type log and we want to exit the API when that moment occurs.